evm/src/trace/utils.rs

```diff
@@ -55,16 +55,17 @@ pub(crate) fn decode_cheatcode_inputs(
             Some(decoded.iter().map(format_token).collect())
         }
         "deriveKey" => Some(vec!["<pk>".to_string()]),
-        "parseJson" | "writeJson" => {
+        "parseJson" | "writeJson" | "keyExists" => {
```

Comment:
Unrelated to this PR, but we should probably add all the `parseJson*` and `serialize*` methods too?

Reply:
yep highly agree, this can be done in another smol pr